pkg/config/setup/system_probe_cws.go

```diff
@@ -118,6 +118,10 @@ func initCWSSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnvAndSetDefault("runtime_security_config.hash_resolver.cache_size", 500)
 	cfg.BindEnvAndSetDefault("runtime_security_config.hash_resolver.replace", map[string]string{})
 
+	// CWS - SysCtl snapshot
+	cfg.BindEnvAndSetDefault("runtime_security_config.sysctl_snapshot.enabled", true)
```

Comment:
are we sure to want to enable it by default?

Reply:
I agree we should first validate everything works as expected before enabling this by default